README.md

```diff
@@ -1,2 +1,197 @@
-# ffc-ce-web
-Future Farming and Countryside programme calculcation engine web
+# FFC Calculation Engine
+
+Boilerplate text.
```

Comment:
Maybe "Boilerplate" should be replaced with something like
This is a demo application for the FFC Calculation Engine